qiskit/transpiler/passes/routing/sabre_swap.py

```diff
@@ -276,15 +275,14 @@ def _reset_qubits_decay(self):
         self.qubits_decay = {k: 1 for k in self.qubits_decay.keys()}
 
     def _successors(self, node, dag):
-        for _, successor, edge_data in dag.edges(node):
+        for _, successor, _ in dag.edges(node):
             if not isinstance(successor, DAGOpNode):
                 continue
-            if isinstance(edge_data, Qubit):
-                yield successor
+            yield successor
```

Comment:
This method can rewritten as:

```python
def _successors(self, node, dag):
    for successor in dag.successors(node):
        if isinstance(successor, DAGOpNode):
            yield succesor
```

It'll be slightly more efficient since we aren't collecting the full edge triplet `(source, target, payload)` and just having retworkx return a list of node payloads. Interestingly there is a `dag.quantum_successors` method which does the previous behavior and is more efficient also since the filtering is done in retworkx.

Reply:
Nice - done in b5b79d2.